openerp-atomfeed-service/src/test/java/org/bahmni/feed/openerp/event/OpenERPCustomerServiceEventWorkerTest.java

```diff
@@ -28,6 +28,8 @@ public class OpenERPCustomerServiceEventWorkerTest {
     private OpenMRSWebClient mockWebClient;
     private String MRSURLPrefix;
 
+    private String endpointURI = "https://client/openmrs/ws/atomfeed/customer/recent";
```

Comment:
Whats the difference between `endpointURL` and `endpointURI`?

Reply:
Typo, will fix it.